Story/FableForest.cs

```diff
@@ -0,0 +1,68 @@
+//cs_include Scripts/CoreBots.cs
+//cs_include Scripts/CoreStory.cs
+using Skua.Core.Interfaces;
+
+public class FableForest
+{
+    public IScriptInterface Bot => IScriptInterface.Instance;
+    public CoreBots Core => CoreBots.Instance;
+    public CoreStory Story = new();
+
+    public void ScriptMain(IScriptInterface bot)
+    {
+        Core.SetOptions();
+
+        StoryLine();
+
+        Core.SetOptions(false);
+    }
+
+    public void StoryLine()
+    {
+        if (Core.isCompletedBefore(3313))
+        {
+            Core.Logger("You have done this storyline");
+            return;
+        }
+
+        Story.PreLoad(this);
+
+        //Cowering Zardman Quests
+        //Axe Marks the Spot 3300
+        Story.KillQuest(3300, "greendragon", "Greenguard Dragon");
+
+        //Get Choppin' 3301
+        Story.MapItemQuest(3301, "fableforest", 2425, 1);
```

Comment:
1 quantatiy is the default , so adding it is redundant

Reply:
Solved